schemas/WebpackOptions.json

```diff
@@ -2487,6 +2487,9 @@
       "properties": {
         "compiler": {
           "description": "Match the child compiler name.",
+          "cli": {
+            "exclude": true
```

Comment:
Whats the motivation of removing all rule related options from the CLI?

The old cli args also allow to specify that e. g. via `--module-bind`.

Is it only of aesthetics? Maybe we can flag them with `"level": "advanced"` instead and omit them in `--help` and only show them in `--help=advanced`

Reply:
> Maybe we can flag them with "level": "advanced" instead and omit them in --help and only show them in --help=advanced

I personally like this idea, I think for now we can support all the flags and we can implement this based on feedback and discussion later.

Reply:
Agree, about `--help=advanced`.

How do you see working with these flags?